build.gradle

```diff
@@ -29,7 +29,15 @@ versioning {
     }
 }
 
-version versioning.ext.makeVersion()
+gradle.projectsEvaluated {
+    tasks.withType(JavaCompile) {
+        // "-Werror" treats any warning (such as those generated by -Xlint:deprecation or -Xlint:unchecked)
+        // as an error, causing the build to fail.
```

Comment:
Is this comment up to date? I don't see `-Werror` anywhere else in the changeset.